src/mixins/filing-template-mixin.ts

```diff
@@ -290,18 +292,27 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
         identifier: this.getEntitySnapshot.businessInfo.identifier,
         legalName: this.getEntitySnapshot.businessInfo.legalName,
         legalType: this.getEntitySnapshot.businessInfo.legalType,
-        nrNumber: this.getEntitySnapshot.businessInfo.nrNumber,
-        associationType: this.getAssociationType
+        nrNumber: this.getEntitySnapshot.businessInfo.nrNumber
       },
       specialResolution: {
-        ...this.getcreateResolution
+        ...this.getCreateResolution
+      },
+      alteration: {
```

Comment:
This looks a bit strange: an alteration object next to the special resolution object. I would have expected one to be nested under the other (or the alteration properties under special resolution?), unless there's a good reason why it's like this.

Reply:
Looks like it was it's own type of filing:

https://github.com/bcgov/business-schemas/blob/6b2268753999371b8731ef6a95218570c4b25412/src/registry_schemas/schemas/filing.json#L69

Reply:
So is `alteration` left over from when the code was cut & pasted? Or does the API require several top-level objects like this?

Unfortunately our schemas don't cover this case; they mostly cover individual objects [like Special Resolution itself]()https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/special_resolution.json).

Reply:
Discussed and resolved in next conversation.